nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-core/src/test/java/org/nuxeo/ecm/directory/core/CoreDirectoryInit.java

```diff
@@ -84,6 +91,15 @@ public void populate(CoreSession session) {
         user1.setProperty("schema1", "bar", "bar1");
         session.saveDocument(user1);
 
+        // Creates SHA1 passwords for unit test
+        String ENC_PWD_USERSHA1 = PasswordHelper.hashPassword(DOC_PWD_USERSHA1, PasswordHelper.SSHA);
```

Comment:
It's better to hardcode the value and not depend on the implementation of `PasswordHelper.hashPassword` in the test. In which case you can actually use a real constant `public static final ENC_PWD_USERSHA1`.